Filters/python/MonoXFilter_cfi.py

```diff
@@ -9,6 +9,6 @@
                              saveWlv = cms.bool(True),
                              saveZll = cms.bool(True),
                              savePho = cms.bool(True),
-                             minU = cms.double(175),
+                             minU = cms.double(0),
```

Comment:
Wait, I don't think this is how we should do it. This turns off the recoil filter for *all* jet collections. This configuration should always be set to 175 (i.e. it's not supposed to be configurable, but I didn't want to hardcode it in C++). Right now, we're running the MonoXFilter (with a threshold of zero), and the FatJetsFiller is checking the output of the filter, which is always going to be true. We should instead do:

1. Run the filter at 175
2. Configure the jet collections to either use or not use the filter [1]

Also @maierbenedikt, we wanted to revisit which jet collections we want (a) no filter on and (b) at all. I think it's time to drop CHS CA15 altogether. We want no filter on (at least) puppi AK8. I propose not computing substructure on CHS AK8 and applying the recoil filter on puppi CA15. Any objections? In summary:

1. CHS CA15 - drop
2. Puppi CA15 - `computeSubstructure='recoil'`
3. CHS AK8 - `computeSubstructure='never'`
4. Puppi AK8 - `computeSubstructure='always'`

[1] https://github.com/PandaPhysics/PandaProd/blob/master/Producer/python/panda_cfi.py#L94